guides/v2.1/coding-standards/technical-guidelines.md

```diff
@@ -697,6 +697,121 @@ class SampleEventObserverThatModifiesInputs
 {:start="14.2"}
 14.2. Events used SHOULD be observed as specifically as possible. A `global` subscription to an event SHOULD NOT be used when the area impacted is just `frontend`.
 
+## 15. XML configuration
+
+15.1 Attributes MUST be used for the configuration values that will not be extended in the future by adding child nodes or new attributes.
+
+15.2 Nodes of the same type that repeat multiple times MUST be wrapped in a grouping node.
+
+15.3 Unnecessarily grouping nodes MUST be avoided.
+
```

Comment:
The left and right columns in the table are identical, except for the comment. Is this intentional?